backend/migrations/versions/af0437dab922__add_dataallManaged_flag.py

```diff
@@ -0,0 +1,24 @@
+"""_add_dataallManaged_flag
+
+Revision ID: af0437dab922
+Revises: f6cd4ba7dd8d
+Create Date: 2024-02-15 10:42:06.833990
+
```

Comment:
We need to make sure that this migration script runs after the one in PR for alembic and keep a consistent alembic migration history

Reply:
I merged migrations